eclass/gnome2.eclass

```diff
@@ -214,7 +214,9 @@ gnome2_pkg_postinst() {
 		gnome2_schemas_update
 	fi
 	gnome2_scrollkeeper_update
-	gnome2_gdk_pixbuf_update
+	if [[ -n ${GNOME2_ECLASS_GDK_PIXBUF_LOADERS} ]]; then
```

Comment:
I don't get why we're doing this actually instead of removing it entirely?

Reply:
To preserve the existing behavior, i.e. not have all `gnome2.eclass` ebuilds call it unnecessarily. Not saying it's a huge overhead but it could be noticeable, especially when installing from binary packages, given that there are actually only a handful of packages installing pixbuf loaders.